tests/test_sparse_embeddings.py

```diff
@@ -0,0 +1,35 @@
+from fastembed.sparse.sparse_text_embedding import SparseTextEmbedding
+
+CANONICAL_COLUMN_VALUES = {
+    "prithvida/SPLADE_PP_en_v1": [
```

Comment:
If we're returning indices and values instead of binary vector, we should test for that as well?

Reply:
added tests